tests/ui/undocumented_unsafe_blocks.rs

```diff
@@ -472,6 +472,19 @@ mod unsafe_impl_invalid_comment {
     unsafe impl Interference for () {}
 }
 
+mod unsafe_items_invalid_comment {
```

Comment:
Can this be split into another file, for example `tests/ui/unnecessary_safety_comment.rs`?